src/components/contents/ProfileCard.tsx

```diff
@@ -29,7 +29,7 @@ const ProfileCard = ({
           <AvatarFallback>{name.substring(0, 2).toUpperCase()}</AvatarFallback>
         </Avatar>
         <div className="flex-start w-full overflow-hidden flex-col gap-1">
-          <h2 className="text-2xl font-semibold text-gray-700 dark:text-zinc-200">
+          <h2 className="lg:text-2xl font-semibold text-gray-700 dark:text-zinc-200 text-1xl">
```

Comment:
In tailwind there are no one classname that is `text-1xl`
I think it will be `text-xl`